### PR TITLE
testing/shc: new aport

### PR DESCRIPTION
**Pull request by mrbitsdcf**
https://neurobin.org/projects/softwares/unix/shc/
A generic shell script compiler.

### CODE REVIEW

File: testing/shc/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributor: MrBiTs <mrbits@mrbits.com.br>
+# Maintainer: MrBiTs <mrbits@mrbits.com.br>
+pkgname="shc"
+pkgver="4.0.3"
+pkgrel=0
+pkgdesc="A generic shell script compiler."
+url="https://neurobin.org/projects/softwares/unix/shc/"
+arch="all"
+license="GPL"
```
Comment (maxice8) [inline testing/shc/APKBUILD]: The license field needs to use a SPDX license identifier

File: testing/shc/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+depends=""
```
Comment (maxice8) [inline testing/shc/APKBUILD]: empty variables can be removed for brevity

File: testing/shc/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+makedepends=""
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/mrbitsdcf/shc/archive/${pkgver}.tar.gz"
```
Comment (maxice8) [inline testing/shc/APKBUILD]: Optional fix: Unnecessary usage of braces

File: testing/shc/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/shc/APKBUILD]: builddir is using the default value, can be removed

File: testing/shc/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	make
+}
+
+check() {
+	# Replace with proper check command(s)
+	:
+}
+
+package() {
+	cd "$builddir"
```
Comment (maxice8) [inline testing/shc/APKBUILD]: no need for this, this is done automatically

File: testing/shc/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	./configure
+	make
+}
+
+check() {
+	# Replace with proper check command(s)
+	:
```
Comment (maxice8) [inline testing/shc/APKBUILD]: Replace with proper check command(s)

Or remove this function and add options="!check" with a comment explaining why test is disabled

**Comment by maxice8** [comment]
Please squash the commits into a single one

File: testing/shc/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	./configure
+	make
+}
+
+package() {
+	install -v -m 0755 -d $pkgdir/bin/
+	install -v -m 0755 $builddir/src/shc $pkgdir/bin/
+}
+
+sha512sums="78fbab26c652b3c3a41401e044ae69776fbdf5d28736c55bcf111f4261f863f6d97d81e87ad9288faf9c9a3299c1453dcca9ffd9d9ea04a1ff2b7299257e8cf7  shc-4.0.3.tar.gz
```
Comment (Ikke) [inline testing/shc/APKBUILD]: I've fixed it locally, but you are missing a closing `"` here.

**Comment by Ikke** [comment]
> Please squash the commits into a single one

Note that you can also do this yourself when it's obvious that one commit just fixes another commit. Then you don't have to wait for someone to fix it before you can apply it.

**Pull request closed without merging**